### PR TITLE
Tests 112

### PR DESCRIPTION
**Pull request by hannesdiedrich**
## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=TESTS-112
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

**Review by mrsaemir — APPROVED** [review]
LGTM

File: src/gsy_e/setup/strategy_tests/future_market_strategy.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    return area
+
+
+def get_var_from_env(var_name: str, default_val: float):
+    """Read variable from env and parse as string"""
+    var = os.environ.get(var_name)
+    if var is not None:
+        return float(var)
+    return default_val
```
Comment (spyrostz) [inline src/gsy_e/setup/strategy_tests/future_market_strategy.py:97]: I assume that you did not use this instead:
```
def get_var_from_env(var_name: str, default_val: float):
    """Read variable from env and parse as string"""
    var = float(os.environ.get(var_name, default_val) or default_val)
```
in order to cover the case that the variable has a value of 0 (in which case you do not want to fall back to the default value)?

**Review by spyrostz — APPROVED** [review]
One clarification question, LGTM already

File: src/gsy_e/setup/strategy_tests/future_market_strategy.py (hunk 1 above)
Comment (hannesdiedrich) [reply src/gsy_e/setup/strategy_tests/future_market_strategy.py:97]: I did not see this part until now. The reasoning you describe make sense.

**Pull request merged**